lib/modules/platform/utils/pr-body.ts

```diff
@@ -1,23 +1,27 @@
-const re = new RegExp(`### Release Notes.*### Configuration`, 'ms');
+const re = new RegExp(`(.*### Release Notes)(.*)### Configuration(.*)`, 'ms');
```

Comment:
Use named capture groups and use `regEx` from `util/regex`

Reply:
Hi @viceice 
I used capture groups 
`const re = regEx(`(?<preNotes>.*### Release Notes)(?<releaseNotes>.*)### Configuration(?<postNotes>.*)`,'ms');`
but when executing `re.exec(input)` it returns null 
and when using the same pattern with new RegExp
`const re =new RegExp(`(?<preNotes>.*### Release Notes)(?<releaseNotes>.*)### Configuration(?<postNotes>.*)`,'ms');` 
it works fine.
should I change the pattern?

Reply:
I think the `m` flag is wrong. it causes `^` and `$` to match start and end of line, so can you test without `m` flag and only use `s` flag?

Reply:
@viceice I removed `m` flag and it doesn't work (`re.exec(input)` returns null).